doc/sphinx/source/esmvalcore/recipe.inc

```diff
@@ -24,12 +24,38 @@ The documentation section includes:
 - A list of scientific references
 - the project or projects associated with the recipe.
 
+For example, please see the documentation section from the recipe:
+recipe_ocean_amoc.yml.
+
+.. code-block:: yaml
+
+    documentation:
+      description: |
+        Recipe to produce time series figures of the derived variable, the
+        Atlantic meriodinal overturning circulation (AMOC).
+        This recipe also produces transect figures of the stream functions for
+        the years 2001-2004.
+
+      authors:
+        - demo_le
+
+      maintainer:
+        - demo_le
+
+      references:
+        - demora2018gmd
+
+      projects:
+        - ukesm
+
 Note that the author, project, and references will need to be included in the
 config-references.yml file. The author name uses the format:
-last name (clipped after 4 letters) underscore first name (clipped to two letters)
-and all letters are lowercase. 
-For instance, Mickey Mouse would be: mous_mi.
-This is unlikely to be the same as the github user name.
+last name (clipped after 4 letters) underscore first name (clipped to two
```

Comment:
Actually, the author name does not have to be in this format, it can be anything the user likes. This is a leftover from version 1.

Reply:
Same comment twice, see below.